tool/cli.js

```diff
@@ -83,6 +83,43 @@ program
     })
   )
 
+  .command("fix-redirects", "Consolidate/fix redirects")
+  .argument("<locale...>", "Locale", {
+    default: DEFAULT_LOCALE,
+    validator: [...VALID_LOCALES.values(), ...VALID_LOCALES.keys()],
+  })
+  .action(
+    tryOrExit(({ args, logger }) => {
+      const { locale } = args;
+      for (const l of locale) {
+        Redirect.add(l.toLowerCase(), []);
+        logger.info(chalk.green(`Fixed ${l}`));
+      }
+    })
+  )
+
+  .command("add-redirect", "Add a new redirect")
```

Comment:
What's up with this? `caporal` is run from a `/Users/peterbe/dev/MOZILLA/MDN/yari/node_modules/@caporal/core/dist/index.js` which is 1 line. Because of that, the whole traceback is filling up my whole terminal screen with compressed JS code. Try it:

```
▶ yarn tool add-redirect "péter " "Основные_Фигуры"
```

I have this vague memory that we used to NOT show the whole traceback unless you added the `--verbose` or something. 
In these cases, it would be nice to capture thrown errors and not include a traceback. Perhaps this is unrelated to the PR at hand, but still a good thing to worry about.

Reply:
What the heck! If run this one, I get a more sensible output from the CLI:

```
▶ yarn tool add-redirect "/fr/docs/péter " "Основные_Фигуры"
yarn run v1.22.10
$ node tool/cli.js add-redirect '/fr/docs/péter ' Основные_Фигуры

error: The from-URL is expected to be /$locale/docs/

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Besides, the error message is `error: The from-URL is expected to be /$locale/docs/` but if you look, my from URL has a `/fr/docs/...`. 
So the error is wrong. I suspect the error should be complaining about that trailing single whitespace in the from-URL.

Reply:
Actually, it's probably not the trailing whitespace. What did I do to deserve this??:

```
▶ yarn tool add-redirect "/fr/docs/péter" "Основные_Фигуры"
yarn run v1.22.10
$ node tool/cli.js add-redirect /fr/docs/péter Основные_Фигуры

error: The from-URL is expected to be /$locale/docs/

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
It because "Основные_Фигуры" doesn't start with `/<locale>/docs/` the _from_ in the error message is misleading. That's not a new issue. I'll remove the _from_ in the error message but everything else should be another PR.

Reply:
But shouldn't that automatically and correctly be taken care of by the validators? ...for each argument. Then caporal should report which one is incorrect.